docker/Dockerfile

```diff
@@ -36,8 +36,7 @@ RUN NPROC=${BUILD_CONCURRENCY:-$(nproc)} && \
     make -j${NPROC} install && \
     cd ../profiles && \
     cp -r * /opt && \
-    strip /usr/local/bin/* && \
-    rm -rf /src /usr/local/lib/libosrm*
```

Comment:
Hm, but now we don’t remove /src as well. do we need that too? /src contains the whole repo contents as I remember, I.e. it will make resulting image quite big.

WDYT?

I am traveling at the moment and reviewing it from my phone’s screen, so could miss something :)

Reply:
Good catch, didn't see that at all :sweat_smile: `src` folder is irrelevant you're right.

Reply:
done:)

Reply:
although `/src` wouldn't even be copied over into the release image (it's multi-stage), but it'd probably still make the intermediate image quite a bit bigger